Devnet examples

camrossi opened the pull request:
This PR adds the code for the demos used in the `DEVNET-2210` Session in Cisco Live Melbourne

thenodon commented:
@camrossi this looks excellent! The file examples/DEVNET-2210/aci-exporter-deployment.yaml is a config map for k8s. Can you also provide one that if you just running aci-exporter in a not k8s env and name it `config.yaml`.

camrossi commented:
> @camrossi this looks excellent! The file examples/DEVNET-2210/aci-exporter-deployment.yaml is a config map for k8s. Can you also provide one that if you just running aci-exporter in a not k8s env and name it `config.yaml`.

I just did another commit when I added a standalone config and also fixed a typo
